README.md

```diff
@@ -1,4 +1,52 @@
 # android-shoppe-demo
 Android Client for the example "Shoppe" app
 
-_Stay tuned!_
+Android Shoppe Demo is reference application that provides tips and tricks to using [FullStory](https://www.fullstory.com/) on [Native Mobile](https://www.fullstory.com/mobile-apps/) Android.
+
+## Getting started
+
+To apply FullStory Android plugin, you'll need [Android Studio](https://developer.android.com/studio). You'll then download or clone this repo to your desired directory.
+
+To open the project, use "File > Open" in Android Studio and selece either "java" or "kotlin"(coming soon) folder.
+
+## Adding FullStory to the app
+
+If you have [signed up](https://www.fullstory.com/plans/) for FullStory, add to your `local.properties` file the following:
+```
+org=<your org id>
+enabledVariants = all
+```
+Your orgId can be found on the Settings page.  It'll be next to the `window['_fs_org']` variable in the snippet.  For more information, see [Install your recording snippet](https://help.fullstory.com/hc/en-us/articles/360020828233#Install).
+
+Unless otherwise instructed, you can remove the follwoing line from your app level `build.gradle`
```

Comment:
> Unless otherwise instructed

Where would this instruction come from? Can you you leave this off? 

> You can remove the follwoing line from your app level `build.gradle`